src/test/java/stepdefinitions/StepDefinitions.java

```diff
@@ -55,12 +56,48 @@ public void iContactValori()throws Throwable{
         assertTrue(contactUsPage.isTitleVisible());
     }
 
-    @Then( "^the contact us form is available$" )
+    @Then("^the contact us form is available$")
     public void theContactUsFormIsAvailable() throws Throwable {
         //TODO IMPLEMENT THIS
         //Verify the contact form fields are available.
-        throw new PendingException();
+
+        //FirstName
+        ContactUsPage contactUsFirstName = new ContactUsPage(driver);
+        contactUsFirstName.write_ContactUsFirstName("Ramazan");
```

Comment:
Je vult nu ook de waardes in vanuit de methode. Dat is niet erg onderhoudbaar aangezien je dan dus voor andere waardes een nieuw stuk code zou moeten schrijven. Probeer dat probleem eens te omzeilen. (hint: Google eens naar cucumber example outline)